data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -0,0 +1,520 @@
+                sandbox_dataset = check_and_create_sandbox_dataset(
+                    date_row.project_id, date_row.dataset_id)
+
+                # Create queries based on type of date field
+                if pd.isnull(date_row.date_column):
+                    sandbox_query = SANDBOX_QUERY_END_DATE.render(
+                        project=date_row.project_id,
+                        sandbox_dataset=sandbox_dataset,
+                        intermediary_table=ticket_number + '_' + date_row.table,
+                        dataset=date_row.dataset_id,
+                        table=date_row.table,
+                        pid=pid,
+                        deactivated_pids_project=pids_project_id,
+                        deactivated_pids_dataset=pids_dataset_id,
+                        deactivated_pids_table=pids_table,
+                        end_date_column=date_row.end_date_column,
+                        start_date_column=date_row.start_date_column)
+                    clean_query = CLEAN_QUERY_END_DATE.render(
+                        project=date_row.project_id,
+                        dataset=date_row.dataset_id,
+                        table=date_row.table,
+                        pid=pid,
+                        deactivated_pids_project=pids_project_id,
+                        deactivated_pids_dataset=pids_dataset_id,
+                        deactivated_pids_table=pids_table,
+                        end_date_column=date_row.end_date_column,
+                        start_date_column=date_row.start_date_column)
+                else:
+                    sandbox_query = SANDBOX_QUERY_DATE.render(
+                        project=date_row.project_id,
+                        sandbox_dataset=sandbox_dataset,
+                        intermediary_table=ticket_number + '_' + date_row.table,
+                        dataset=date_row.dataset_id,
+                        table=date_row.table,
+                        pid=pid,
+                        deactivated_pids_project=pids_project_id,
+                        deactivated_pids_dataset=pids_dataset_id,
+                        deactivated_pids_table=pids_table,
+                        date_column=date_row.date_column)
+                    clean_query = CLEAN_QUERY_DATE.render(
+                        project=date_row.project_id,
+                        dataset=date_row.dataset_id,
+                        table=date_row.table,
+                        pid=pid,
+                        deactivated_pids_project=pids_project_id,
+                        deactivated_pids_dataset=pids_dataset_id,
+                        deactivated_pids_table=pids_table,
+                        date_column=date_row.date_column)
+                queries_list.append({
+                    clean_consts.QUERY: sandbox_query,
+                    clean_consts.DESTINATION_DATASET: sandbox_dataset,
+                    clean_consts.DESTINATION_TABLE: date_row.table,
```

Comment:
@amcgrenera-vumc you will probably want to change this to use a `WRITE_APPEND` disposition.